packages/contracts/ts/deploy.ts

```diff
@@ -147,7 +147,7 @@ export const deployContract = async <T extends BaseContract>(
  * @returns the deployed VkRegistry contract
  */
 export const deployVkRegistry = async (signer?: Signer, quiet = false): Promise<VkRegistry> =>
-  deployContract<VkRegistry>("VkRegistry", signer, quiet);
+  deployContract<VkRegistry>("VkRegistry", signer, quiet, await signer?.getAddress());
```

Comment:
This function deploys the `VKRegistry` in all our codebase so I did not need to change it anywhere else. I am sending the signer address as an argument in the deployment step